.github/workflows/pr.yml

```diff
@@ -38,7 +38,7 @@ jobs:
           output_path: 'results'
           platform_type: terraform
           fail_on: high,medium
-          exclude_queries: '8e94dced-9bcc-4203-8eb7-7e41202b2505,e592a0c5-5bdb-414c-9066-5dba7cdea370' # Excludes 'Auto Scaling Group With No Associated ELB' and 'IAM Access Analyzer Undefined' queries
```

Comment:
Had a look at the exclusion, I feel that it can remain added to the `exclude_queries` list. Have now added a description. It comes from this resource;
```
resource "aws_kms_key" "aurora" {
  enable_key_rotation     = true
  deletion_window_in_days = 7
  tags                    = merge(var.tags, { Name = "${var.name}-db-key", ProtectsSensitiveData = true })
  policy                  = var.kms_key_policy != null ? var.kms_key_policy : data.aws_iam_policy_document.kms_key_policy.json
}
```
Specificallly the policy;
```
KMS Key With Vulnerable Policy, Severity: HIGH, Results: 1
Description: Checks if the policy is vulnerable and needs updating.
```